acceptance/acceptance_test.go

```diff
@@ -76,6 +76,11 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 	cwd, err := os.Getwd()
 	require.NoError(t, err)
 
+	buildDir := filepath.Join(cwd, "build", fmt.Sprintf("%s_%s", runtime.GOOS, runtime.GOARCH))
```

Comment:
We don't have to scope the build to the os and arch right? Any single machine only has one of those.

Reply:
We do - if you use Windows VM mounted to the same directory, builds conflict othewise.